qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Filters for vacancies
+
+Background:
+Given User is on https://career.quantori.com/ru/positions
+And vacancies available with Security and Compliance Competence
+And vanacies available with Linux Techology
+
+Scenario: Drop-down of Competence and Technologies
+When User clicks the fields Competence or Technologies
```

Comment:
Yo should write it the following way:
When user clicks on <name> dropdown
Then <line> should be visible

Examples:
| name | line |
| Competence | Security and Compliance |
|  Technologies |  Linux |